spec/system/admin/adjustments_spec.rb

```diff
@@ -26,11 +26,11 @@
   before do
     order.finalize!
     create(:check_payment, order: order, amount: order.total)
+    login_as_admin_and_visit spree.admin_orders_path
   end
 
   it "adding taxed adjustments to an order" do
     # When I go to the adjustments page for the order
-    login_as_admin_and_visit spree.admin_orders_path
     page.find('td.actions a.icon-edit').click
     click_link 'Adjustments'
```

Comment:
Idea for another improvement: We don't need to click through all these pages in each scenario. It wastes time. One test should check that the page is reachable this way but other specs can go straight to the page we are testing.